library/src/main/java/net/alhazmy13/hijridatepicker/date/hijri/HijriDatePickerDialog.java

```diff
@@ -1025,7 +1027,7 @@ public UmmalquraCalendar getEndDate() {
         if (mMaxDate != null) return mMaxDate;
         UmmalquraCalendar output = new UmmalquraCalendar(getTimeZone(), getLocale());
         output.set(UmmalquraCalendar.YEAR, mMaxYear);
-        output.set(UmmalquraCalendar.DAY_OF_MONTH, 31);
+        output.set(UmmalquraCalendar.DAY_OF_MONTH, 29);
```

Comment:
Not sure why you change this to 29?

Reply:
after updating ummalqura the values 31 and 30 caused a crash